### ISSUE TITLE
"sketch has been modified" message cannot be dismissed.

### ISSUE THREAD

**Issue by hillmark** [processing/processing#3898]
When opening 2.x sketches a modal dialog is presented, which states: "Your sketch has been modified externally. Would you like to reload the sketch?"

This dialog cannot be dismissed regardless of the action, and a forced restart is required.

Not a huge problem, but worth noting.

Thanks.

**Comment by benfry** [comment]
What version of Processing are you using? What operating system? Where is the sketch stored?

It has nothing to do with opening a 2.x sketch; it's about your machine and where the files are stored.

**Comment by hillmark** [comment]
I'm running 3.0b7 on 10.10.3 (Yosemite)

Yes, you're right it's about where the file is stored. I was attempting to open an archived sketch from a folder stored on an SD card, which works okay for 2.x

Opening a 2.x sketch from a folder on the local file system is working fine.

**Comment by benfry** [comment]
Ok, so that means https://github.com/processing/processing/issues/3387 is not fixed.

And the dialog box keeps popping up even if you click yes or no?

**Comment by hillmark** [comment]
That's correct, the modal cannot be dismissed by clicking yes or no, hence the forced restart.

**Comment by benfry** [comment]
Can you try quitting Processing, then opening ~/Library/Processing/preferences.txt and changing this line:
`editor.watcher.debug = false`
to 
`editor.watcher.debug = true`
and try running Processing after that? It should print some text to the console about why it thinks the file has changed. Can you report that back here?

**Comment by JakubValtar** [comment]
@hillmark Also could you please send a list of files in your sketch folder with visible extensions? Printscreen of the sketch folder is enough. We think there might be some funny filename causing the problems. Thanks.

**Comment by hillmark** [comment]
I've been able to isolate what appears to be the issue. The external SD in question is ExFAT, which means that when these sketches were originally copied `._` prefixed counterparts were created to as a way to handle metadata unsupported by ExFAT.

I've created an isolated case using one of the example sketches, and found that by adding a `._` prefixed `.pde`, such as `._Foo.pde` file replicates the problem, and `._` files without a `.pde` extension work fine.
 
![screen shot 2015-09-30 at 09 09 01](https://cloud.githubusercontent.com/assets/801177/10187823/f1051dc8-6752-11e5-9337-c3619c63ebd8.png)

@benfry in answer to your request for a console log, here's the output produced:

![screen shot 2015-09-30 at 09 26 34](https://cloud.githubusercontent.com/assets/801177/10188212/7b66223a-6755-11e5-9090-3a3002b67b5e.png)

Thanks.
